renovate.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ],
+  "reviewers": ["team:data-platform-mysql"],
+  "enabledManagers": ["poetry", "pip_requirements", "github-actions", "regex"],
```

Comment:
`poetry` is not really necessary, but might stay if the charm is later made to use it.